CalibCalorimetry/EcalLaserSorting/src/LaserSorter.cc

```diff
@@ -1156,6 +1154,6 @@ void LaserSorter::restoreStreamsOfLumiBlock(int lumiBlock) {
   }
 }
 
-void LaserSorter::beginRun(edm::Run const& run, edm::EventSetup const& es) {
-  //  cout << "Run starts at :" << run.runAuxiliary().beginTime().value() << "\n";
-}
+void LaserSorter::beginRun(edm::Run const& run, edm::EventSetup const& es) {}
+
+void LaserSorter::endRun(edm::Run const& run, edm::EventSetup const& es) {}
```

Comment:
Since these are not doing anything they can be also removed (maybe for the future... they also dont bother too much)